sdk/monitor/monitor-opentelemetry-exporter/src/export/base.ts

```diff
@@ -93,11 +104,25 @@ export abstract class AzureMonitorBaseExporter {
    */
   protected async _exportEnvelopes(envelopes: Envelope[]): Promise<ExportResult> {
     diag.info(`Exporting ${envelopes.length} envelope(s)`);
+    
+    // Shutdown statsbeat if the maximum number of failures is exceeded
+    if (
```

Comment:
You don't want to do any of this checks if isStatsbeatExporter=true

Reply:
Well I can see you check for _statbeatMetrics to exist, maybe just check for boolean isStatsbeatExporter and make it first check

Reply:
This check is removed as a part of other optimizations.